cli/cmd/aws.go

```diff
@@ -49,6 +51,15 @@ func awsDescribeInstances() ([]*lwrunner.AWSRunner, error) {
 	return allRunners, nil
 }
 
+func GetConfig() (aws.Config, error) {
+	cfg, err := config.LoadDefaultConfig(context.Background())
+	if err != nil {
+		return cfg, err
+	}
+
+	return cfg, nil
```

Comment:
Is this different to just `return config.LoadDefaultConfig(context.Background())`?

Reply:
I agree that that's simpler, we can just do that. But, I actually want to change this to not have a `GetConfig` function, and just pass the `cfg` struct down from the toplevel caller. I'll probably do this in a separate PR.

Reply:
I am with @quodlibetor here, if we created a ticket to refactor it later, let's put this back. (My $0.02)

Reply:
Looking at it again, I agree with this. Removed with 16aa209ac553b404c32d9f3e97eeef2c2c560630